valueWithCoords.go

```diff
@@ -1,14 +0,0 @@
-package main
```

Comment:
I have no memory of this place

Reply:
zawsze można cofnąć się po commitach, jak będzie jakiś reusable kod